dbs/watchlist.js

```diff
@@ -0,0 +1,54 @@
+const lock = require('../lib/lock')
+const db = require('./profile-data-db')
+
+class BadParamError extends Error {
+  constructor (msg) {
+    super()
+    this.name = 'BadParamError'
+    this.message = msg
+  }
+}
+
+// exported methods
+// =
+
+exports.addSite = async function (profileId, url, opts) {
+  var release = await lock('watchlist-db')
+  try {
+    // get date for timestamp in seconds floored
+    var ts = (Date.now() / 1000 | 0)
+
+    // check if site already being watched
+    var site = await db.get('SELECT rowid, * from watchlist WHERE profileId = ? AND url = ?', [profileId, url])
+    if (site) {
+      throw "Site already being watched."
```

Comment:
Still wrap in `new Error()` just for hygiene

```suggestion
      throw new Error('Site already being watched.')
```